src/Aspire.Hosting.Azure.AppContainers/ContainerAppEnvironmentContext.cs

```diff
@@ -43,7 +43,7 @@ public async Task<AzureBicepResource> CreateContainerAppAsync(IResource resource
             await context.ProcessResourceAsync(cancellationToken).ConfigureAwait(false);
         }
 
-        var provisioningResource = new AzureContainerAppResource(resource.Name, context.BuildContainerApp, resource)
+        var provisioningResource = new AzureContainerAppResource(resource.Name + "-containerapp", context.BuildContainerApp, resource)
```

Comment:
Why do we need to make this change?

Reply:
Ooooh yeah, we'll want to make sure to document this. I can see it being a bit of failure for people who don't know how the internal map is built.